tests/functional/raw-node/utils/gcpUtils.js

```diff
@@ -0,0 +1,26 @@
+const { makeGcpRequest } = require('./makeRequest');
+
+function gcpRequestRetry(params, retry, callback) {
+    const retryTimeout = {
+        0: 0,
+        1: 1000,
+        2: 2000,
+        3: 4000,
+        4: 8000,
```

Comment:
Why these timeout times?

Reply:
At the moment, they are hardcoded timeouts for exponential backoff. I have been getting 'SlowDown' responses, because GCP has Bucket creation/deletion operations limited to 1 op per 2 seconds.
https://cloud.google.com/storage/docs/exponential-backoff
https://cloud.google.com/storage/quotas